rules.txt

```diff
@@ -277,6 +283,7 @@
 /ethicsboard/pages/default.aspx           301  /departments/board-of-ethics/
 /evacuations/(.*)                         301  /services/safety-emergency-preparedness/evacuations/$1
 /executive_orders/?.*                     301  /executiveorders/
```

Comment:
Can you also update executive_orders to go to the new location? Prevents a double redirect.

Reply:
done

Reply:
Sorry, did you push that change? I'm not seeing it